Riot/Modules/Room/ContextualMenu/ReactionsMenu/ReactionsMenuViewModel.swift

```diff
@@ -195,4 +209,37 @@ final class ReactionsMenuViewModel: ReactionsMenuViewModelType {
             self.react(withReaction: unreaction, selected: false)
         }
     }
+
+    /// reactUsingHack directly sends a `m.reaction` room message instead of using the `/send_relation` api.
+    ///
+    /// TODO: To remove once the feature has landed on matrix.org homeserver
+    ///
+    /// - Parameter reaction: the reaction
+    private func reactUsingHack(withReaction reaction: ReactionsMenuReaction) {
+        print("[ReactionsMenuViewModel] reactUsingHack")
+
+        let reactionContent = [
+            "m.relates_to": [
+                "rel_type": "m.annotation",
+                "event_id": self.eventId,
+                "key": reaction.rawValue]
+        ]
+
+        var nilEvent: MXEvent?
+        let room = self.session.room(withRoomId: self.roomId)
```

Comment:
We could add `guard let room = self.session.room(withRoomId: self.roomId)` at the beginning of the method and ideally call `reactionsMenuViewModel(_, didReactionFailedWithError:, reaction:, isAddReaction:)` in the `else`.